bin/provision-instances

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/python3
+#
+# This script runs a full instance provision over all non-ephemeral instances
+# on OCIM. Running this script requires Django to be initialized - the easiest
+# way to do this is by evaluating the script from within the Django shell by
+# running:
```

Comment:
Ah yes.. a django management command would work better for this, instead of just a standalone script.  Sorry, that was my bad advice to put this into `bin`!